WORKSPACE

```diff
@@ -2,8 +2,8 @@ workspace(name = "com_github_istio_mixer")
 
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "9496d79880a7d55b8e4a96f04688d70a374eaaf4", # Mar 3, 2017 (v0.4.1)
     remote = "https://github.com/bazelbuild/rules_go.git",
+    tag = "0.4.2",
```

Comment:
i thought at some point we decided on using commit SHAs, as tags could change out from under us. Can we point at the commit SHA for 0.4.2 instead?